### ISSUE TITLE
Documentation for automatic data harvesting

### ISSUE THREAD

**Issue by brew** [openspending/openspending#1384]
As part of #1381. Once work is complete on improving the data harvesting and pipeline status dashboard, we should improve documentation for users of the automatic data harvesting pipelines. Currently, [os-data-importers](https://github.com/openspending/os-data-importers) provides a very basic description of functionality, and no guidance for its use, either self-hosted, or how to contact the OpenSpending team to 'enrole' as a supported user. Topics for documentation may include the following, but further topics should be included on this issue as work on #1381 proceeds:

#### General
- how to write a fiscal.source-spec.yaml file
- creating a user on OpenSpending.org to use as `owner` in fiscal.source-spec.yaml

#### OpenSpending.org hosted pipelines
- committing changes to the official source-spec registry
- checking pipeline status
- contacting helpdesk?

#### Self-hosted pipelines
- installing pipelines and optional source-spec registry

**Comment by sandervdwaal** [comment]
@tlacoyodefrijol Could you have a look at this issue please? We'd need to write out first what the user stories are for this and then provide the documentation, either by linking to what we've already got in the README's on Github or by writing separate documentation on eg. the community.openspending.org site.

**Comment by sglavoie** [comment]
**WONTFIX/CLEANUP**: Closing old issue without recent activity (>1.5 years) nor assignee as part of #1478. [GIFT Phase IV](https://github.com/openspending/openspending/projects/1) was closed. Please feel free to re-open if this is still relevant.
